app/services/notification/Notifications.scala

```diff
@@ -0,0 +1,63 @@
+package services.notification
+
+import javax.inject.{Inject, Singleton}
+
+import config.{AMIableConfig, AmiableConfigProvider}
+import org.quartz.CronScheduleBuilder.cronSchedule
+import org.quartz.JobBuilder.newJob
+import org.quartz.{JobKey, TriggerKey}
+import org.quartz.TriggerBuilder.newTrigger
+import org.quartz.impl.StdSchedulerFactory
+import play.api.inject.ApplicationLifecycle
+import play.api.{Configuration, Environment, Logger, Mode}
+
+import scala.concurrent.{ExecutionContext, Future}
+
+@Singleton
+class Notifications @Inject()(
+                               amiableConfigProvider: AmiableConfigProvider,
+                               environment: Environment,
+                               lifecycle: ApplicationLifecycle,
+                               mailClient: AWSMailClient)
+                             (implicit exec: ExecutionContext,
+                              configuration: Configuration) {
```

Comment:
I will integrate the 2 strings used from `Configuration` into `amiableConfigProvider` so that only that is passed around!